projects/schematics/src/collection.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
+    "schematics": {
+        "ng-add": {
+            "description": "Add my library to the project.",
```

Comment:
Shouls the description be 

Add OIDC lib to project?

Reply:
Absolutely.